contxt/cli/commands/iot.py

```diff
@@ -48,6 +47,33 @@ def groupings(clients: Clients, facility_id: int, fields: List[str], sort: str)
     print_table(items=items, keys=fields, sort_by=sort)
 
 
+@iot.command()
+@click.option("--output_id", required=True, type=int, help="Output ID to delete from")
+@click.option(
+    "--field", required=True, type=str, help="The field to delete from. Ex: power.demand, power.usage",
+)
+@click.option(
+    "--interval",
```

Comment:
i know this maps behind the scenes to different tables, but is this option clear to the user what it's doing and why we need it? in order to truly "delete" one data point i would potentially have to call this command for every item in `Window`, otherwise i didn't truly delete it correct?

i would suggest trying to handle this confusing mess for them with needing to pass `--interval`, but unfortunately i don't think we can do anything about it. for example, consider we observed an outlier at `2020-01-01T00:18:01Z`. behind the scenes, the tables store the first observation in a given interval, which may look like:

* raw table: `2020-01-01T00:18:01Z`, `the_outlier` (simple!)
* 1 min table: `2020-01-01T00:18:00Z`, `some_value_that_may_be_the_outlier` (first observation from `00:18:00Z` -> `00:19:00Z`)
* 15 min table: `2020-01-01T00:30Z`, `some_value_that_may_be_the_outlier` (first observation from `00:30:00Z` -> `00:45:00Z`)
* 60 min table: `2020-01-01T00:00Z`, `some_value_that_may_be_the_outlier` (first observation from `00:00:00Z` -> `01:00:00Z`)

but given that time, it is non-deterministic if i need to delete from anything other than the raw table because i have no way to know if it's the time was the first observation in the interval or not. long way of saying, the current design is far from ideal.

Reply:
I agree it's not the best design, but we're just trying to take a small step forward with it. Currently, the exact parameters for these options are provided by a manually run script.